docs/source/python_api/exceptions/mlflow.exceptions.rst

```diff
@@ -0,0 +1,16 @@
+:orphan:
+
+mlflow.exceptions
+=================
+
+.. exception:: mlflow.exceptions.MlflowException(message, error_code=1, **kwargs)
+
+    Generic exception thrown when an MLflow operation fails
```

Comment:
Explicitly override the docstring of `MlflowException` since the current description is targeted at developers constructing the exception rather than callers catching it